File: Language/C/Selectionsort.c
#include<stdio.h>

int main(){
  printf("enter the no. of integers you eant to sort");
  scanf("%d",&n);
  in a[n];
//input for all the numbers
  for(i=0;i<n;++i){
      scanf("%d",&a[i]);
  }
//sorting starts
  for(i=0;i<n;++n){
      for(j=i+1;j<n;++j){
          if(a[j]<a[i]){
          temp=a[i];
          a[i]=a[j];
          a[j]=temp;
          }
      }
  }
//printing the array
  for(i=0;i<n;++i){
      scanf("%d",&a[i]);
  }

return 0;
}

File: Language/Python/Selection-Sort.py
#Selection Sort

def sorte(l):
    for i in range(len(l)-1):
        min=i
        for j in range(i,len(l)):
            if(l[j]<l[min]):
                min=j
        temp=l[i]
        l[i]=l[min]
        l[min]=temp

l=[8,5,3,6,7,2,63,17,23]

sorte(l)

print(l)